### PR TITLE
Fix jest in TestApp

### PR DESCRIPTION
**Pull request by MatkovIvan**

### CODE REVIEW

**Comment by dhei** [comment]
@MatkovIvan, eslint is not happy with mocks in `__mocks__` thus failing Build, please fix it.

**Comment by guperrot** [comment]
@MatkovIvan Let's wait #303 to do that and have test app point at mock files provided by SDK.

File: TestApp/__mocks__/appcenter-crashes.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const Crashes = jest.mock('appcenter-crashes');
```
Comment (guperrot) [inline TestApp/__mocks__/appcenter-crashes.js]: Please catch up with develop and remove those. Might need to revisit the PR entirely now that #303 is merged.

**Comment by guperrot** [comment]
When I run that I have the following:

```

**Pull request merged**